packages/protocol/contracts/L1/v1/V1Proposing.sol

```diff
@@ -58,8 +65,27 @@ library V1Proposing {
             meta.txListHash
```

Comment:
If we allow proposers skip committing blocks when `K_COMMIT_DELAY_CONFIRMATIONS == 0`, this `_calculateCommitHash()` calculation can also be moved into the `if (LibConstants.K_COMMIT_DELAY_CONFIRMATIONS > 0) {}` section below to save some gas?

Reply:
fixed